spec/appliance_console/logfile_configuration_spec.rb

```diff
@@ -94,17 +94,19 @@
       it "stops and starts evm and configures the logfile disk" do
         subject.new_logrotate_count = 3
         subject.disk = double(@spec_name, :size => "9999999", :path => "fake disk")
-  
-        expect(ApplianceConsole::LogicalVolumeManagement).to receive(:new).and_return(double(@spec_name, :setup => true))
+
+        expect(ApplianceConsole::LogicalVolumeManagement)
+          .to receive(:new).and_return(double(@spec_name, :setup => true))
         expect(File).to receive(:executable?).with("/sbin/restorecon").and_return(true)
         expect(AwesomeSpawn).to receive(:run!).with('/sbin/restorecon -R -v /var/www/miq/vmdb/log')
-        expect(FileUtils).to receive(:mkdir_p).with("#{ApplianceConsole::LogfileConfiguration::LOGFILE_DIRECTORY}/apache")
+        expect(FileUtils)
+          .to receive(:mkdir_p).with("#{ApplianceConsole::LogfileConfiguration::LOGFILE_DIRECTORY}/apache")
```

Comment:
I would prefer this split to happen later so the first line isn't so short.
Maybe:
```ruby
expect(FileUtils).to receive(:mkdir_p)
  .with("#{ApplianceConsole::LogfileConfiguration::LOGFILE_DIRECTORY}/apache")
```